data/mods/Aftershock/items/ammo/needle.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "id": "3x35dart",
+    "type": "AMMO",
+    "name": { "str": "GG Stinger 3x35mm flechette" },
+    "description": "The Gibson Gear Stinger flechette is a magnetically propelled dart roughly half the length of the average human thumb.   ",
+    "weight": "7 g",
+    "volume": "105 ml",
+    "price": 1125,
+    "material": [ "steel" ],
+    "symbol": "=",
+    "color": "green",
+    "count": 100,
+    "stack_size": 75,
+    "ammo_type": "3x35",
+    "range": 10,
+    "damage": { "damage_type": "bullet", "amount": 8, "armor_penetration": 20 },
+    "dispersion": 60,
+    "recoil": 40,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "3x35frangible",
+    "copy-from": "3x35dart",
+    "type": "AMMO",
+    "name": { "str": "GG Splinter 3x35mm flechette" },
+    "description": "Gibson Gear Splinter flechettes are made with magnetic ceramics that shatter upon impact and dig deeply into flesh, as a consequence of this change these rounds lose almost all of their armor piercing.",
+    "weight": "6 g",
+    "price": 1688,
+    "material": [ "ceramic" ],
+    "relative": { "range": -1, "damage": { "damage_type": "bullet", "amount": 14, "armor_penetration": 2 }, "dispersion": 20 },
```

Comment:
```suggestion
    "relative": { "range": -1, "damage": { "damage_type": "bullet", "amount": 14, "armor_penetration": -18 }, "dispersion": 20 },
```

I presume you meant to decrease AP, not increase it :P.
![image](https://user-images.githubusercontent.com/99621099/158909496-1d4dcb0b-f021-4e81-a83b-b06af274f4bc.png)

Reply:
Good catch!